src/client/clang_command.cc

```diff
@@ -100,6 +100,11 @@ bool ClangCommand::FillFlags(base::proto::Flags* flags,
              arg->getOption().matches(OPT_D) ||
              arg->getOption().matches(OPT_I)) {
       arg->render(arg_list_, non_cached_list);
+    } else if (arg->getOption().matches(OPT_include_pch) ||
+               arg->getOption().matches(OPT_include_pth)) {
+      auto* included_file = flags->add_included_files();
+      *included_file = arg->getValue();
```

Comment:
What are the rules for path calculation for a value of this option? Is it relative to current folder or input file? Or does it use include paths?